mobx/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ authors:
 homepage: https://github.com/mobxjs/mobx.dart
 
 environment:
-  sdk: '>=2.2.0 <3.0.0'
+  sdk: '>=2.7.0 <3.0.0'
```

Comment:
Can you educate me on how rxdart has done this? We are forcing everyone to move up to dart 2.7, which is fine since extensions can't be used otherwise. Curious how other packages are dealing with this?

Reply:
In the case of rxdart, they proposed using extensions to replace the internal wrappers when the feature was in BETA. When Flutter 1.12 with dart 2.7 support was released, they updated the package generating Breaking Changes, but it was necessary to use these new features as extensions and raise the lib threshold.

I believe that with MobX the story will be different as it will be idol for Flutter as developers are always very close to the stable version and always update.

I have doubts as to other platforms that use Dart, in this environment I do not know how this is applied.

Reply:
I am happy to upgrade if folks are always on latest Flutter. @shyndman, @rrousselGit any insights I should know before I bump the dart version to **2.7**?

Reply:
I think it should be fine. People can always reference an earlier version if needed.

Reply:
Great, let's do this with min as 2.7